script.js

```diff
@@ -1,12 +1,23 @@
-//You can edit ALL of the code here
-function setup() {
-  const allEpisodes = getAllEpisodes();
-  makePageForEpisodes(allEpisodes);
+// Setup initial view (series list)
+async function setup() {
+  let seriesList = await fetchData("https://api.tvmaze.com/shows?page=0");
+  seriesList = seriesList.sort((a, b) =>
+    a.name > b.name ? 1 : b.name > a.name ? -1 : 0
+  );
+  loadSeriesView(seriesList);
+  history.pushState(null, null, "series");
+  document.getElementById("alphabetic").checked = true;
 }
 
-function makePageForEpisodes(episodeList) {
-  const rootElem = document.getElementById("root");
-  rootElem.textContent = `Got ${episodeList.length} episode(s)`;
+// Fetch data from api
+async function fetchData(url) {
+  let result;
```

Comment:
You have a mix of promises (then) and async/await.  try to just use one of the other